SDWebImage/Core/SDImageCacheDefine.m

```diff
@@ -13,6 +13,37 @@
 #import "UIImage+Metadata.h"
 #import "SDInternalMacros.h"
 
+static NSArray<NSString *>* GetKnownContextOptions(void) {
```

Comment:
Hack for 5.13. In 5.14 I will remove this and use new `SDWebImageContextImageDecodeOptions` instead.

See more in PR